src/components/Trade/hooks/useSwapper/useSwapper.tsx

```diff
@@ -321,25 +322,22 @@ export const useSwapper = () => {
         to: assetReference,
       })
 
-      const { networkFeeCryptoBaseUnit, ...fees } = await getFees({
-        accountNumber: activeQuote.steps[0].accountNumber,
+      const buildCustomTxInput = await createBuildCustomTxInput({
         adapter,
         to: assetReference,
         value,
         data,
         wallet,
+        accountNumber: activeQuote.steps[0].accountNumber,
```

Comment:
We might want a `selectFirstHopAccountNumber` selector.

Reply:
this is the old implementation so that swapper doesnt exist ;)